packages/editor-ui/src/components/mixins/pushConnection.ts

```diff
@@ -260,7 +260,8 @@ export const pushConnection = mixins(
 						this.$titleSet(workflow.name as string, 'ERROR');
 
 						if (
-							runDataExecuted.data.resultData.error!.name === 'ExpressionError' &&
+							runDataExecuted.data.resultData.error &&
+							runDataExecuted.data.resultData.error.name === 'ExpressionError' &&
```

Comment:
nit: You can now use `runDataExecuted.data.resultData.error?.name === 'ExpressionError'`.

Reply:
oh ya thanks..